src/core/scene/a-scene.js

```diff
@@ -118,8 +124,15 @@ var AScene = module.exports = registerElement('a-scene', {
     addBehavior: {
       value: function (behavior) {
         var behaviors = this.behaviors;
-        if (behaviors.indexOf(behavior) !== -1) { return; }
-        behaviors.push(behavior);
+        for (var b in behaviors) {
```

Comment:
We usually use `Object.keys().forEach` pattern for object enumeration, which I read to be a bit faster.